Applications/Calculator/CalculatorWindow.gml

```diff
@@ -0,0 +1,272 @@
+@GUI::Widget {
+    fixed_width: 254
+    fixed_height: 213
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10,0,10,0]
+    }
```

Comment:
In Libraries/LibGUI/FontPickerDialog.gml (lines 5, 6 and 7), this is written with the following style; let's be consistent.  
```suggestion
    layout: @GUI::VerticalBoxLayout {
        margins: [10, 0, 10, 0]
    }
```